apps/mobile/src/features/browser/approver-sheet/stx/call-contract/hooks.ts

```diff
@@ -0,0 +1,57 @@
+import { useCallback } from 'react';
+
+import { useStacksSigners } from '@/store/keychains/stacks/stacks-keychains.read';
+import { bytesToHex } from '@stacks/common';
+
+import { RpcRequest, stxCallContract } from '@leather.io/rpc';
+import {
+  TransactionTypes,
+  generateStacksUnsignedTransaction,
+  getStacksAssetStringParts,
+} from '@leather.io/stacks';
+import { useOnMount } from '@leather.io/ui/native';
+
+import { StxNormalizedParams } from '../utils';
+
+interface UseCallContractTxHex {
+  message: RpcRequest<typeof stxCallContract>;
+  normalizedParams: StxNormalizedParams;
```

Comment:
What does normalized mean here, and how is it a relevant detail for this component?